lib/cdo/contact_rollups.rb

```diff
@@ -46,6 +46,16 @@ class ContactRollups
     query_timeout: MAX_EXECUTION_TIME_SEC
   )
 
+  def self.mysql_multi_connection
+    # return a connection with the MULTI_STATEMENTS flag set that allows multiple statements in one DB call
+    sequel_connect(
+      CDO.pegasus_reporting_db_writer,
+      CDO.pegasus_reporting_db_writer,
```

Comment:
Just to make sure this is intentional, the first 2 params are the same and both are db_writer?

Reply:
Yes, just like in the provisioning of [`PEGASUS_REPORTING_DB_WRITER`](https://github.com/code-dot-org/code-dot-org/blob/5a56399cda9abef6f7e5b0669619c37ab273fbf8/lib/cdo/contact_rollups.rb#L36)